samples/template_ble/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -1,11 +1,28 @@
 /*
- * Copyright (c) 2022 Nordic Semiconductor ASA
+ * Copyright (c) 2024 Nordic Semiconductor ASA
```

Comment:
Code from future, maybe we could make a template to autofill the header for new files, this would solve the issue of incorrect year in the copyright